recipes/install.rb

```diff
@@ -686,9 +666,12 @@
   action :create
 end
 
-["convert-ipython-notebook.sh", "anaconda-rsync.sh", "zip-hdfs-files.sh", "zip-background.sh",
-  "unzip-background.sh", "anaconda-command-ssh.sh", "conda-command-ssh.sh", "tensorboard.sh", "tensorboard-launch.sh", 
-  "tensorboard-cleanup.sh", "condasearch.sh", "pipsearch.sh", "list_environment.sh"].each do |script|
+["anaconda-rsync.sh", "zip-hdfs-files.sh", "zip-background.sh",
+ "unzip-background.sh", "anaconda-command-ssh.sh", "conda-command-ssh.sh",
+ "tensorboard-launch.sh", "tensorboard-cleanup.sh", "condasearch.sh",
```

Comment:
I don't think we need all these scripts anymore - like `anaconda-rsync`, `anaconda-command-ssh`, `conda-command-ssh`, "list_environment.sh", "jupyter-kill.sh", "jupyter-launch.sh",
 "tfserving-kill.sh", "sklearn_serving-launch.sh", "sklearn_serving-kill.sh"

Reply:
I think a bunch of these scripts is used by kagent. As I have not touched kagent in this work (cleaning it up will be a future jira) I have preferred to let the removal of these scripts to the kagent work.